Source/Noesis.Javascript/JavascriptInterop.cpp

```diff
@@ -271,9 +267,10 @@ JavascriptInterop::WrapObject(System::Object^ iObject)
 
 	if (context != nullptr)
 	{
-		Handle<ObjectTemplate> templ = context->GetObjectWrapperTemplate();
+		Handle<FunctionTemplate> templ = context->GetObjectWrapperConstructorTemplate(iObject->GetType());

```

Comment:
Probably simply returning the `InstanceTemplate` is enough, then this can stay `ObjectTemplate` and the context simply caches the `ObjectTemplate` instead of the full `FunctionTemplate`.

Reply:
Ah, but if we add the `RegisterConstructor` the `FunctionTemplate` return value is probably more flexible.